configuration/blockly/rules-blockly-standard-ext.md

```diff
@@ -92,6 +112,13 @@ A color of r=255, g=255, b=255 would therefore result into #ffffff preceded by a
 Note that typically RGB values are not used within openHAB for color states but instead the Color-Type expects a HSB Type.
 See the special hsb-conversion block below.
 
+- pick a color to return the value as an RGB type
```

Comment:
Seems this list of Blockly’s abilities regarding colors is missing a header sentence.
Probably add something like: “Blockly supports the following use cases:”